frontend/src/hooks/api/actions/useProjectApi/useProjectApi.ts

```diff
@@ -68,6 +82,25 @@ const useProjectApi = () => {
         }
     };
 
+    const editProjectSettings = async (
```

Comment:
We should use orval types here and throughout this logic.

Reply:
Done but with problems with the generation